src/domain/orders/details/returns/index.js

```diff
@@ -136,219 +149,328 @@ const ReturnMenu = ({ order, onReturn, onDismiss, toaster }) => {
+                              />
+                            </div>
+                            <div className="inter-small-regular text-grey-50 flex flex-col ml-4">
+                              <span>
+                                <span className="text-grey-90">
+                                  {item.title}
+                                </span>{" "}
+                                test
+                              </span>
+                              <span>{item.variant.title}</span>
+                            </div>
+                          </div>
+                        </Table.Cell>
+                        <Table.Cell className="text-right w-32 pr-8">
+                          {toReturn.includes(item.id) ? (
+                            <div className="flex w-full text-right justify-end text-grey-50 ">
+                              <span
+                                onClick={() => handleQuantity(-1, item)}
+                                className="w-5 h-5 flex items-center justify-center rounded cursor-pointer hover:bg-grey-20 mr-2"
+                              >
+                                <MinusIcon size={16} />
+                              </span>
+                              <span>{quantities[item.id] || ""}</span>
+                              <span
+                                onClick={() => handleQuantity(1, item)}
+                                className={clsx(
+                                  "w-5 h-5 flex items-center justify-center rounded cursor-pointer hover:bg-grey-20 ml-2"
+                                )}
+                              >
+                                <PlusIcon size={16} />
+                              </span>
+                            </div>
+                          ) : (
+                            <span className="text-grey-40">
+                              {item.quantity - item.returned_quantity}
+                            </span>
+                          )}
+                        </Table.Cell>
+                        <Table.Cell className="text-right">
+                          {(item.refundable / 100).toFixed(2)}
+                        </Table.Cell>
+                        <Table.Cell className="text-right text-grey-40 pr-1">
+                          {order.currency_code.toUpperCase()}
+                        </Table.Cell>
+                      </Table.Row>
+                      {checked && (
+                        <Table.Row className="last:border-b-0 hover:bg-grey-0">
+                          <Table.Cell colspan={5}>
+                            <div className="w-full flex justify-end">
+                              <Button
+                                variant="ghost"
+                                size="small"
+                                className="border border-grey-20"
+                              >
+                                Select Reason
+                              </Button>
```

Comment:
doesn't do anything, that's not part of this PR right?

Reply:
it is now 😄